orchagent/mirrororch.cpp

```diff
@@ -565,11 +600,7 @@ bool MirrorOrch::deactivateSession(const string& name, MirrorEntry& session)
     }
 
     session.status = false;
-
-    if (!setSessionState(name, session))
-    {
-        throw runtime_error("Failed to test session state");
-    }
+    setSessionStatus(name, session);
```

Comment:
For the deactivation part, it is still okay to set all the related information other than the activate or not information.